CHANGELOG.md

```diff
@@ -2,6 +2,58 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.5.0] - 19.08.2020
+
+:warning: **Breaking change:** :warning:  
+Please update to this version as soon as the mainnet coordinator got upgraded.  
+Instructions on how to update to this version will be posted in due time.  
+The old HORNET versions won't be functional within the mainnet anymore!  
+
+### Added 
+
+    - White-Flag
+    - Conflicting badge to the transaction explorer
+    - Weighted uniform random tipselection 
+    - Adaptive heaviest branch tipselection
+    - COO heaviest branch tipselection
+    - conf_trytes (confirmed trytes) ZMQ topic 
+    - conf_trytes (confirmed trytes) MQTT topic
+    - Optional powsrv.io PoW support
+    - Neighbors count to heartbeat message
+    - LMI and neighbor counts to dashboard
+    - getTipInfo API call
+
+### Changed
+
+    - Return false for conflicting tx in the getInclusionStates web api call
+    - Request tx from all neighbors that could have the data
+    - Bump protocol feature set for whiteflag
+    - Reconnect websocket on failue
+    - Update to Go 1.15
+
+### Fixed
+
+    - Missing byte to trytes conversion in some error messages
+    - Make code more testable
+    - Adding autopeered neighbors as static neighbors
+    - CTPS calculation
+    - Revalidation 
+    - Pruning
+    - High dashboard traffic
+    - Misc page not shown if spammer is enabled
+
+### Removed
+
+    - Unused defaults from config[_comnet, _devnet].json
+    - Graph plugin
+    - Monitor plugin
+    - Legacy gossip protocol
+    - Genesis tx special case
+
+### Config file changes
+
+Please use the new config.json and transfer values from your current config.json over to the new one, as a lot of keys have changed or got removed (instead of mutating your current one).
+
 ## [0.5.0-rc13] - 14.08.2020
```

Comment:
Remove all RCs?

Reply:
As they are partly incomplete, this may be a good idea, yes :+1: